gobblin-dataset-config/src/main/java/gobblin/dataset/config/ConfigAccessor.java

```diff
@@ -0,0 +1,22 @@
+package gobblin.dataset.config;
+
+import java.util.*;
+import com.typesafe.config.Config;
+
+public class ConfigAccessor {
+
+  private String configVersion;
+  private ConfigStore configStore;
+  
+  public Config getConfig(String urn){
```

Comment:
I assume there should be a constructor that takes a configVersion and configStore parameters.